### ISSUE TITLE
Blog post design

### ISSUE THREAD

**Issue by crawlem** [spacebetweentherapy/website#36]
Intro above the brown line, rest of the post below it, followed by the image at the end of the post. Then a brown section with a list of previous posts.

**Comment by crawlem** [comment]
Done, ready for review. If you're happy, you can close this issue ticket.

**Comment by richoster** [comment]
Yep, this layout looks good. Two tweaks (which I don't know if I should keep in this ticket or create a new one?) would be 1) to reduce the width of the text and image, so it is the same width as the 'about' and 'therapy' pages. 2) Add a 'back' button so it's easy to get back to the journal page. I'm sure you've made it so I can do this, so maybe I should just have a try. Also maybe it's not needed as they could press back on browsers or click one of the links below, but my first impression was that it might be good.

**Comment by crawlem** [comment]
I've reduced the width of the text column and updated the image sizes to match. The CMS now restricts blog post images to the correct size.

In terms of back button I think we should pick this up under #41 as a general "journal navigation" ticket.

**Comment by richoster** [comment]
perfect, sounds good
